src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -7237,3 +7237,47 @@ def test_aks_create_with_enable_network_observability(self, resource_group, reso
         # delete
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus', preserve_default_location=True)
+    def test_aks_create_with_custom_headers(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=75441342&view=results) to validate the change.